### PR TITLE
Add mobile nav component

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

**Comment by asasmith** [comment]
@kcbreuil do you have a specific question on this one? I thought you were going to try adding the classes we discussed this morning and see if that resolves the issue

**Comment by kcbreuil** [comment]
@asasmith the placement of the button between 600-700px bothers me. I can't get it to move without destroying the placement of the hamburger nav div. Thoughts>

File: web/src/components/header.js [hunk 1]
```diff
@@ -56,12 +57,23 @@ const nav = [
...
+    const handleNavToggle = () => {
+        setIsMobileNavOpen(!isMobileNavOpen);
+    };
+
     return (
-        <nav className="fixed z-[1]">
-            <div className="container flex justify-between mx-24">
-                <Link to="/" className="pt-11">
+        <nav className="fixed py-8 top-0 transition-colors w-full z-10">
+            <div className="container flex justify-between mx-auto text-white">
```
Comment (asasmith) [inline web/src/components/header.js:68]: I don't think you need `mx-auto` here

**Review by asasmith — CHANGES_REQUESTED** [review]
The close button needs a little work.

Screen Shot 2021-06-07 at 8 41 30 AM

File: web/src/components/navigation/mobileNav.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    const firstSpanStyles = isMobileNavOpen ? '-rotate-45' : 'rotate-0';
+
+    const secondSpanStyles = isMobileNavOpen
+        ? 'opacity-0 w-0'
+        : 'opacity-100 w-full';
+
+    const thirdSpanStyles = isMobileNavOpen ? 'rotate-45' : 'rotate-0';
+    const firstSpanLength = isMobileNavOpen ? 'w-10' : 'w-11';
+    const secondSpanLength = isMobileNavOpen ? 'w-10' : 'w-8';
+    const thirdSpanLength = isMobileNavOpen ? 'w-10' : 'w-5';
```
Comment (asasmith) [inline web/src/components/navigation/mobileNav.js:25]: Can these declarations be combined with the `*Styles` declarations?

File: web/src/components/navigation/mobileNav.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    return (
+        <>
+            <button
+                className="absolute flex flex-col justify-between items-end lg:hidden h-8 w-10 right-0 top-1/2 transform -translate-y-1/2 -translate-x-4 z-10"
+                type="button"
+                onClick={handleNavToggle}
+            >
+                <span
+                    className={`h-1 rounded-full transform-gpu w-11 transition transition-all duration-300 origin-right ${firstSpanStyles} ${colorStyle} ${firstSpanLength}`}
```
Comment (asasmith) [inline web/src/components/navigation/mobileNav.js:35]: remove `w-11`

**Review by asasmith — CHANGES_REQUESTED** [review]

File: web/src/components/navigation/mobileNav.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+export default function MobileNav({
+    navItems,
+    isMobileNavOpen,
+    handleNavToggle,
+}) {
+    const colorStyle = isMobileNavOpen ? 'bg-black' : 'bg-white';
```
Comment (asasmith) [inline web/src/components/navigation/mobileNav.js:10]: can these be combined into the other span styles too?

**Pull request merged**